### PR TITLE
Add ability to change the island name

### PR DESCRIPTION
**Pull request by MertUnverdi**

### CODE REVIEW

**Review by PeachesMLG — APPROVED** [review]

File: src/main/java/com/iridium/iridiumskyblock/commands/RenameCommand.java [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+
+public class RenameCommand extends Command {
+
+    public RenameCommand() {
+        super(Collections.singletonList("rename"), "Change your the island name", "", true);
```
Comment (dlsf) [inline src/main/java/com/iridium/iridiumskyblock/commands/RenameCommand.java:20]: "Change the name of your island"

File: src/main/java/com/iridium/iridiumskyblock/commands/RenameCommand.java [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+     * Executes the command for the specified {@link CommandSender} with the provided arguments.
+     * Not called when the command execution was invalid (no permission, no player or command disabled).
+     * Allows a user to change the island name.
+     *
+     * @param sender The CommandSender which executes this command
+     * @param args   The arguments used with this command. They contain the sub-command
+     */
+    @Override
+    public void execute(CommandSender sender, String[] args) {
+        Player p = (Player) sender;
```
Comment (dlsf) [inline src/main/java/com/iridium/iridiumskyblock/commands/RenameCommand.java:33]: Don't use such short variable names

File: src/main/java/com/iridium/iridiumskyblock/commands/RenameCommand.java [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+        if (args.length != 2) {
```
Comment (dlsf) [inline src/main/java/com/iridium/iridiumskyblock/commands/RenameCommand.java:34]: If you want to support names that have more than one word, this should be == 1

File: src/main/java/com/iridium/iridiumskyblock/commands/RenameCommand.java [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+     *
+     * @param commandSender The CommandSender which tries to tab-complete
+     * @param command       The command
+     * @param label         The label of the command
+     * @param args          The arguments already provided by the sender
+     * @return The list of tab completions for this command
+     */
+    @Override
+    public List<String> onTabComplete(CommandSender commandSender, org.bukkit.command.Command command, String label, String[] args) {
+
```
Comment (dlsf) [inline src/main/java/com/iridium/iridiumskyblock/commands/RenameCommand.java:81]: Remove this empty line

File: src/main/java/com/iridium/iridiumskyblock/configs/Messages.java [hunk 5]
```diff
@@ -65,6 +65,10 @@
     public String userDemotedPlayer = "%prefix% &7%promoter% has demoted %player% to %rank%.";
     public String transferredOwnership = "%prefix% &7%oldowner% has transferred Island ownership to %newowner%.";
     public String islandValue = "%prefix% &b&l * &7%rank%: &b%value%";
+    public String cannotChangeName = "%prefix% &7You cannot change the island name!";
+    public String islandNameTooLong = "%prefix% %name% is too long for the island name, The maximum size is %max_length%";
```
Comment (dlsf) [inline src/main/java/com/iridium/iridiumskyblock/configs/Messages.java:69]: "%prefix% %name% is too long for the island name, the maximum length is %max_length%"

File: src/main/java/com/iridium/iridiumskyblock/configs/Messages.java [hunk 6]
```diff
@@ -65,6 +65,10 @@
...
+    public String islandNameTooShort = "%prefix% %name% is too short for the island name, The minimum size is %min_length%";
```
Comment (dlsf) [inline src/main/java/com/iridium/iridiumskyblock/configs/Messages.java:70]: "%prefix% %name% is too long for the island name, the minimum length is %min_length%"

**Review by dlsf — CHANGES_REQUESTED** [review]

File: src/main/java/com/iridium/iridiumskyblock/commands/RenameCommand.java (hunk 1 above)
Comment (dlsf) [reply src/main/java/com/iridium/iridiumskyblock/commands/RenameCommand.java:20]: Oh, it sounded really bad but I think it's also correct. It's up to you

Comment (PeachesMLG) [reply src/main/java/com/iridium/iridiumskyblock/commands/RenameCommand.java:20]: Hes doesnt make sence, use the one das sent

Comment (PeachesMLG) [reply src/main/java/com/iridium/iridiumskyblock/commands/RenameCommand.java:20]: or make it `Change your island name`

**Review by dlsf — APPROVED** [review]

**Review by dlsf — APPROVED** [review]

**Pull request merged**